src/EditorFeatures/Core/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -90,7 +91,7 @@ private bool TryExecuteCommand(int caretPosition, Document document, CommandExec
             // a presenter that can accept streamed results.
             if (streamingService != null && streamingPresenter != null)
             {
-                StreamingFindReferences(document, caretPosition, streamingService, streamingPresenter);
+                _ = StreamingFindReferencesAsync(document, caretPosition, streamingService, streamingPresenter);
```

Comment:
Why are we dropping a Task here?

Reply:
➡️ This is part of the VSTHRD110 follow-up item mentioned in the original description. I also filed https://github.com/Microsoft/vs-threading/issues/345 to ensure we don't accidentally fail to address this.